src/main/java/cz/student/zvedamrance/TaskManager.java

```diff
@@ -1,9 +1,119 @@
-package cz.student.zvedamrance;
+package org.example;
+import org.apache.commons.lang3.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 
 public class TaskManager {
+    private static final File file = new File("tasks.csv");
+    private static String[][] tasks;
+    private static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        loadTasks();
 
-    public static void main(final String[] args) {
+        while (true) {
+            System.out.println(ConsoleColors.BLUE + "\nPlease select an option: " + ConsoleColors.RESET);
+            for (String option : OPTIONS) {
+                System.out.println("-> " + option);
+            }
+
+            final String input = scanner.nextLine();
+            switch (input) {
+                case "add" -> add();
+                case "remove" -> remove();
+                case "list" -> list();
+                case "exit" -> exit();
+                default -> System.out.println(ConsoleColors.RED + "Please select a correct option." + ConsoleColors.RESET);
+            }
+        }
+    }
 
+    public static void add() {
+        tasks = Arrays.copyOf(tasks, tasks.length + 1);
+        System.out.println("Please enter the name of the task you would like to add:");
+        final String name = scanner.nextLine();
+        System.out.println("Please enter the date (format YYYY-MM-DD):");
+        final String date = scanner.nextLine();
+        System.out.println("Please enter if the task is done or not (true/false):");
+        final String isDone = scanner.nextLine();
+        tasks[tasks.length - 1] = new String[] {name, date, isDone};
     }
 
+    public static void remove() {
+        System.out.println("Please, enter the number of the task you would like to remove:");
+        int number = -1;
+        while (true) {
+            final String input = scanner.nextLine();
+            try{
+                number = Integer.parseInt(input);
```

Comment:
CLEAN-CODE: lepsi pojmenovani jedna se o index v poli, number je dosti obecne.

```suggestion
                final int index = Integer.parseInt(input);
```

Reply:
opraveno